File: prob05/prob5.rb
def rand_print(num)
	array = []
	if num > 40
		(40..num).each do |x|
			if x % 2 != 0
				array << x
			end
		end
	else
		(0..40).each do |x|
			if x % 2 != 0
				array << x
			end
		end
	end
	return array
end

rand_num = rand(1..100)
puts "The number is " + rand_num.to_s + "\n"
print rand_print(rand_num)

File: prob05/prob5.py
from random import randrange

def rand_print(num):
	array = []
	if num > 40:
		for x in range(40,num+1):
			if x % 2 != 0:
				array.append(x)
	else:
		for x in range(0,40+1):
			if x % 2 != 0:
				array.append(x)
	return array

rand_num = randrange(100)
print "The number is " + str(rand_num)
print rand_print(rand_num)